src/components/organisms/pages/page_school/School.jsx

```diff
@@ -0,0 +1,46 @@
+// import { Button, Icon } from "/src/components/Organisms/pages/index.js";
+
+import { Button } from "/src/components/molecules/Button";
+import { Icon } from "/src/components/molecules/Icon";
+import { FiLock } from "react-icons/fi";
+import image from "/src/assets/teacher.jpg";
+
+export const School = ({ showCardShadow }) => {
+  const shadowClasses = showCardShadow
+    ? "lg:shadow-lg lg:rounded-3xl lg:p-16"
+    : "";
+  return (
+    <div
+      className={`flex w-[310px] flex-col items-center lg:mx-16 lg:h-[770px] lg:w-full ${shadowClasses}`}
+    >
+      <h1 className="mb-6 text-3xl font-bold">Class books</h1>
+
+      <div className="flex flex-wrap justify-center gap-6 overflow-y-scroll lg:w-[800px]">
+        {[1, 2, 3, 4, 5, 6, 7, 8, 9].map((num) => {
```

Comment:
Nice work, although you could implement a TEACHERS object, to leave the logic implemented by each teacher, i.e. TEACHERS = { object teacher }, with different data but with the same picture to avoid duplication.

Reply:
You're right, I changed it to an array constant TEACHER with object having the information for each teacher.